plugins/repository-hdfs/src/main/java/org/elasticsearch/repositories/hdfs/HdfsRepository.java

```diff
@@ -51,6 +49,8 @@
 
 public final class HdfsRepository extends BlobStoreRepository {
 
+    private static final Logger LOGGER = Logger.getLogger(HdfsRepository.class);
```

Comment:
Why are we using the old Log4j API here? I understand that Hadoop needs it, but we can use the new Log4j API here, and our Log4j API to acquire a logger, no?